MatrixSDK/Data/MXRoomState.m

```diff
@@ -112,13 +112,34 @@ + (void)loadRoomStateFromStore:(id<MXStore>)store
                matrixSession:(MXSession *)matrixSession
                   onComplete:(void (^)(MXRoomState *roomState))onComplete
 {
+    NSString *loadEventId = [NSUUID UUID].UUIDString;
```

Comment:
This is a bit of an experiment, adding a unique ID to all logs that are logically related but may not follow right after each other in time (e.g. making API calls). In this case if multiple room states are being loaded, the ID will disambiguate them. If this approach turns out to be useful, we could abstract this identification into a better API.

Reply:
Sounds helpful to me. I wonder if having `eventId` in the name could cause some confusion until reading through the code though?

Reply:
Yea was wondering about this too. Renamed for now, and I'll think about the abstraction next.